### DISCUSSION TITLE
z_xxx_yy.zip files in custom location

### DISCUSSION THREAD

**Discussion by Falcon0703** [kubilus1/autoortho#217] [Q&A]
I'm really running low on C:/ drive so downloaded and saved the zipped files to a custom location. However, as soon as I start installation, it again starts downloading all the zipped files in C drive. Is there a way to make it utilize the files that I already have downloaded and saved in my location?

**Comment by TinkerNZ** [comment]
You need to and can set the Custom Scenery path in the AO menu before you run the install of the base packs. The same approach applies to the Cache Directory Path. This can get rather large over time and you may choose to place on another drive path.
Important you set these alternate paths 'before' you install the base packs and run AO for the first time.
